admin/upgrade/3.x-to-4.x/index.rst

```diff
@@ -0,0 +1,104 @@
+.. _geonode-upgrade-3.x-4.x:
+
+========================================
+Upgrade Process from GeoNode 3.x to 4.x
+========================================
+
+The purpose of this document is to detail the process that we will implement to upgrade GeoNode from version 3.x to version 4.x. This document is proposed to ensure that the upgrade process is efficient and understandable for everyone. It is important to note that this guide is specifically for a GeoNode installation running on Docker.
+
+The objective of this document is for everyone to understand the processes and steps necessary to carry out the migration.
+
+Table of Contents
+
+* Migration Procedure
+* Possible Issues
+
+Version Migration Procedure
+===========================
+
+1. Download the New Version
+---------------------------
+
+With the following command, we will download the new version of GeoNode from the official source or the corresponding repository:
+
+::
+
+    https://github.com/GeoNode/geonode.git -b 4x
+
+It is recommended to do this in a separate directory to keep the previous version intact in case of need.
+
+* Directory of the previous version:
+
+::
+
+    my/work/dir/geonode3
+
+* Directory of the new version:
+
+::
+
+    my/work/dir/geonode4
+
+2. Stop the Current GeoNode
+---------------------------
+
+Run the following command in the directory where the current GeoNode is deployed to stop the associated Docker containers:
+
+::
+
+    docker-compose down
```

Comment:
"`docker-compose`" is for old versions of Docker. Newer versions use "`docker compose`", because compose has become a core plugin (and command).
I suggest to use the latest version of the command inside this documentation.

Reply:
done